tensorflow/workspace.bzl

```diff
@@ -129,11 +129,11 @@ def tf_workspace(path_prefix="", tf_repo_name=""):
   native.new_http_archive(
       name = "eigen_archive",
       urls = [
-          "http://bazel-mirror.storage.googleapis.com/bitbucket.org/eigen/eigen/get/f3a22f35b044.tar.gz",
-          "https://bitbucket.org/eigen/eigen/get/f3a22f35b044.tar.gz",
+          #"http://bazel-mirror.storage.googleapis.com/bitbucket.org/eigen/eigen/get/e805e4c4a75c.tar.gz",
+          "https://bitbucket.org/mehdi_goli/opencl/get/aea492f0c623.tar.gz",
```

Comment:
This is an intermediate commit - we will remove it once our current Eigen pull request gets accepted.